extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -382,7 +391,14 @@ export class Visibility {
       state[LAST_CHANGE_ENTRY] = change;
 
       // Update states and check if all conditions are satisfied
-      if (this.updateCounters_(visible, listener, shouldBeVisible)) {
+      const conditionsMet =
+          this.updateCounters_(visible, listener, shouldBeVisible);
+
+      if (!shouldBeVisible) {
+        // For "hidden" trigger, only update state, don't trigger.
```

Comment:
The way I understand this, it looks like the hidden conditions will never trigger when the page is in background and the visibility conditions start meeting. Is that correct?